cognite/client/utils/_concurrency.py

```diff
@@ -237,6 +234,22 @@ def get_executor(max_workers: int) -> TaskExecutor:
     return executor
 
 
+def get_as_completed_fn(
```

Comment:
Why this rather than the static methods on the implementation?

Reply:
So I can definitely revert it 👍

Reply:
Reverted in #c44b555